src/twitch/TwitchHelper.ts

```diff
@@ -113,6 +86,32 @@ export default class TwitchHelper {
       });
   }
 
+  private async requestWrapper(streamers: string[]): Promise<StreamData> {
+    return new Promise(
+      (
+        resolve: (data: StreamData) => void,
+        reject: (e: Error) => void
+      ): void => {
+        request(
+          {
+            host: 'api.twitch.tv',
+            path: `/helix/streams?${TwitchHelper.generateUrl(streamers)}`,
+            headers: {
+              'Client-ID': Settings.getSettings()['twitch-id'],
+              Authorization: `Bearer ${this.#accessToken}`,
+            },
+          },
+          (res: IncomingMessage): void => {
+            let data: string = '';
+            res.on('error', reject);
+            res.on('data', (chunk: Buffer): string => (data += chunk));
+            res.on('end', () => resolve(JSON.parse(data)));
+          }
+        ).end(TwitchHelper.generateUrl(streamers));
+      }
+    );
+  }
```

Comment:
What about moving this to `common.ts` and using `request` asynchronously continuously?